Neos.Fusion/Classes/FusionObjects/ActionUriImplementation.php

```diff
@@ -195,13 +232,26 @@ public function evaluate()
         }
 
         try {
-            return $uriBuilder->uriFor(
+            $arguments = $this->getArguments();
+            $routingArguments = $this->getRoutingArguments();
+            if ($arguments && $routingArguments) {
+                throw new RuntimeException('Neos.Fusion:ActionUri does not allow to combine "arguments" and "routingArguments"', 1665431866);
+            }
```

Comment:
Nice :D

Thanks!

Reply:
Was a clean suggestion and removes all questions regarding merging of arguments